lib/http/request.js

```diff
@@ -241,7 +241,6 @@ class HttpRequest extends EventEmitter {
       return;
     }
 
-    this.emit('error', err);
```

Comment:
But why remove this error event?

Reply:
the error event is getting emitted twice for a request.
https://github.com/nightwatchjs/nightwatch/blob/da83b77d28d07583ee6a65dac3dbbeb67fa4908a/lib/http/request.js#L143

Reply:
I think it's better to remove the line with `this.proxyEvents(this.httpRequest, ['error']);`

Reply:
Done 🟢